pkg/ottl/ottlfuncs/README.md

```diff
@@ -586,6 +587,23 @@ Examples:
 
 - `Concat(["HTTP method is: ", span.attributes["http.method"]], "")`
 
+### Contains
+
+`Contains(target, item)`
+
+The Contains function checks if an item is present in a given slice `target`. It returns true if the `item` is found, and false otherwise.
```

Comment:
It would be nice to link/mention here that this function compares the slice values using the same OTTL [comparison rules](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/ottl/LANGUAGE.md#comparison-rules).